### PR TITLE
Add a Getting Started tutorial for the PHP CDA SDK.

### PR DESCRIPTION
**Pull request by realityking**
For your reviewing pleasure @grncdr.

Please don't merge it yet, I'll do so once the SDK is public.

### CODE REVIEW

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+In Contentful, we separate content between Entries, which contain your data and relationships with other
+content or images, and Assets, which represent static content, like images, and are served as files. You can read more
+about that in our [Data Model Concepts guide][3].
+
+In this section we'll address Entries. Assets will be addressed on a later section.
+
+With the client already created, all that's left to do is to start consuming the data from the API.
+
+With the client already created, all that's left to do is to start consuming the data from the API.
```
Comment (grncdr) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: Duplicate line here.

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+This tutorial will walk you through your first steps in using Contentful within your PHP application.
+With only a few simple steps you will be able to access the content you create on Contentful.
+
+## Installation
+
+The easiest way to install the Contentful PHP SDK is to use [Composer][2]. If you've downloaded
+`composer.phar`, the SDK can be installed by executing:
+
+~~~ bash
+php composer.phar install contentful/contentful.php
```
Comment (grncdr) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: The package name should be `contentful/contentful` instead, the `.php` is useful for the repository name but kind of redundant here.

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+$catEntriesByDate = $client->getEntries($query);
+~~~
+
+### Using your Entry
+
+Once we've got our entry, you can access the content it holds trough getter methods:
+
+~~~ php
+echo $cat.getName(); // "Nyan Cat"
+echo "I have $cat.getLives() lives"; // "I have 1337 lives"
```
Comment (grncdr) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: Pretty sure these dots should be `->`s

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+$query = new \Contentful\Delivery\Query;
+$query->orderBy('sys.createdAt');
+$assets = $client.getAssets($query);
+~~~
+
+Once you have an asset, you can access it's metadata and an URL for the actual file:
+
+~~~ php
+echo $asset->getName(); // "Nyan Cat"
+echo $asset->getName()->getFile()->getUrl(); // "//images.contentful.com/cfexampleapi/4gp6taAwW4CmSgumq2ekUm/9da0cd1936871b8d72343e895a00d611/Nyan_cat_250px_frame.png"
```
Comment (grncdr) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: the `->getName()` shouldn't be here.

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+~~~ php
+$query = new \Contentful\Delivery\Query;
+$query->setContentType('cat')
+      ->orderBy('sys.createdAt');
+$catEntriesByDate = $client->getEntries($query);
+~~~
+
+### Using your Entry
+
+Once we've got our entry, you can access the content it holds trough getter methods:
```
Comment (dlitvakb) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: There is a Typo here

```diff
-trough
+through
```

File: docs/tutorials/php/getting-started-with-contentful-and-php.md (hunk 3 above)
Comment (dlitvakb) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: There are a few more below these ones

**Comment by realityking** [comment]
I've addressed all comments.

**Comment by hollensteiner** [comment]
Php tags are necessary to enable proper syntax highlighting, added them.

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+~~~ php
+<?php
+require_once 'vendor/autoload.php';
+?>
+~~~
+
+## Setting up your Contentful Client
+
+Once the SDK is installed you can start using it by creating a `Client`. To do so you need the Id of the space you'd
```
Comment (neonichu) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: I would use `ID` or `id`

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+?>
+~~~
+
+## Getting your content
+
+In Contentful, we separate content between Entries, which contain your data and relationships with other
+content or images, and Assets, which represent static content, like images, and are served as files. You can read more
+about that in our [Data Model Concepts guide][3].
+
+In this section we'll address Entries. Assets will be addressed on a later section.
```
Comment (neonichu) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: "in a later section"

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+$query = new \Contentful\Delivery\Query;
+$query->setContentType('cat')
+      ->orderBy('sys.createdAt');
+$catEntriesByDate = $client->getEntries($query);
+?>
+~~~
+
+### Using your Entry
+
+Once we've got our entry, you can access the content it holds through getter methods:
```
Comment (neonichu) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: I would keep the subject the same through one sentence, so either "we" or "you"

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+~~~
+
+~~~ php
+<?php
+$entryId = 'nyancat';
+$entry = $client->getEntry($entryId);
+?>
+~~~
+
+To specify more [complex queries][4] you can use the query builder
```
Comment (neonichu) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: missing `:`

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+
+~~~ php
+<?php
+$query = new \Contentful\Delivery\Query;
+$query->orderBy('sys.createdAt');
+$assets = $client->getAssets($query);
+?>
+~~~
+
+Once you have an asset, you can access it's metadata and an URL for the actual file:
```
Comment (neonichu) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: > its

Comment (realityking) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: :see_no_evil: I hate making that one

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+To use the example space it would look like this:
+
+~~~ php
+<?php
+$client = new \Contentful\Delivery\Client('b4c0n73n7fu1', 'cfexampleapi');
+~~~
+
+## Getting your content
+
+In Contentful, we separate content between Entries, which contain your data and relationships with other
```
Comment (mariobodemann) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: If you capitalize `Entries`, you should probably also write `relationships` with a capital `R`, or the other way arround.

Comment (neonichu) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: I think the convention is to capitalise Contentful specific terms, which *Entries* are but not *relationships*.

File: docs/tutorials/php/getting-started-with-contentful-and-php.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+<?php
+require_once 'vendor/autoload.php';
+~~~
+
+## Setting up your Contentful Client
+
+Once the SDK is installed you can start using it by creating a `Client`. To do so you need the ID of the space you'd
+like to use and an access token for this space, both of which can be obtained through the Contentful UI.
+
+To use the example space it would look like this:
```
Comment (dlitvakb) [inline docs/tutorials/php/getting-started-with-contentful-and-php.md]: Should probably change it to:

```diff
- To use the example space it would look like this:
+ For this tutorial we'll use the Example Space:
```

**Comment by realityking** [comment]
Merged despite test failures. Th failures were only in the API Blueprints and unrelated to this pull request.

**Pull request merged**